### PR TITLE
705 Casper SDK Format

### PR DESCRIPTION
**Pull request by ACStoneCL**
### Related links

[Adjust Casper SDK Standard formattting #705](https://github.com/casper-network/docs/issues/705)

### Changes

Rather than change around the entire format, I've created a new document that appears before the SDK Standard in the sidebar. This document directs people based on their level of interaction with the JSON-RPC.

### Notes

Ran locally without issue.

### CODE REVIEW

File: source/docs/casper/dapp-dev-guide/json-rpc.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Most programming languages that compile to [WebAssembly (Wasm)](/glossary/W/#webassembly) are compatible with Casper. If there is not currently an SDK for your preferred programming language, you are free to build your own [Casper SDK Standard](/dapp-dev-guide/sdkspec/introduction/) compliant SDK.
+
+The standard outlines necessary available endpoints and their included types, both for fully-featured SDKs and those of a more narrow scope. Guidance for Casper SDK developers can be found [here](/dapp-dev-guide/sdkspec/guidance/).
+
+## Interacting Directly with the JSON-RPC
+
+For advanced users that wish to interact directly with the JSON-RPC API, they are free to do so using various tools like [Postman](https://www.postman.com/).
+
+The Casper JSON-RPC API is fully compatible with the [JSON-RPC 2.0 Specification](https://www.jsonrpc.org/specification).
```
Comment (ipopescu) [inline source/docs/casper/dapp-dev-guide/json-rpc.md:21]: I suggest adding a few more details at the end of this section (after line 21):

Casper nodes provide the RPC schema on port `8888`, followed by `rpc-schema`:  

```sh
<HOST:8888>/rpc-schema 
```

To see an example, navigate to http://65.21.235.219:8888/rpc-schema in your browser.

The Casper client subcommand `list-rpcs` provides all currently supported RPCs. Here is an example of running the Casper client to list RPCs:

```sh
casper-client list-rpcs -n http://65.21.235.219:7777
```

Comment (ipopescu) [reply source/docs/casper/dapp-dev-guide/json-rpc.md:21]: Apologies, I could not use the suggestion tag here, as the formatting was getting mangled.

**Review by ipopescu — COMMENTED** [review]
The structure and content are great. I added a few more details that might be useful.

**Comment by ACStoneCL** [comment]
@ipopescu I folded in your suggestions, and after an embarrassing amount of commits, I fixed the build, haha.

**Review by ipopescu — APPROVED** [review]
Looks good :)

**Review by darthsiroftardis — APPROVED** [review]

**Pull request merged**